src/main/java/org/spongepowered/api/block/BlockType.java

```diff
@@ -121,4 +123,18 @@
      */
     float getEmittedLight();
 
+    /**
+     * Gets if the block is able to be gathered using the silk touch
+     * enchantment.
+     *
+     * @return If the block can be harvested with silk touch
+     */
+    boolean canHarvestWithSilk();
+
+    /**
+     * Gets the equivalent {@link ItemBlock} for this BlockType.
+     *
+     * @return The equivalent {@link ItemBlock}, if available
+     */
+    Optional<ItemBlock> getHeldItem();
```

Comment:
Interesting.

Very VERY few blocks in the base game have no ItemBlock. BlockAir is one off the top of my head that does not. Just an observation.